distro/depends/pft

```diff
@@ -1 +1 @@
-libnuma1
+
```

Comment:
does this change impact other rootfs? similar questions to other removes in distro/depends